### PR TITLE
Fix typos and spelling errors found by codespell

### PR DESCRIPTION
**Pull request by garrett**
Most of these changes are in comments. I'll note where it affects the code inline.

### CODE REVIEW

File: test/check-machines-migrate [hunk 1]
```diff
@@ -90,47 +90,47 @@ class TestMachinesMigration(VirtualMachinesCase):
     def testSharedStorageMigration(self):
         self._testMigrationGeneric(False, False, None)
 
-    @skipImage('RHEL doesnt provide support for copy-storage migration', "rhel-8-4", "rhel-8-5", "rhel-8-6", "rhel-9-0", "centos-9-stream")
+    @skipImage('RHEL does not provide support for copy-storage migration', "rhel-8-4", "rhel-8-5", "rhel-8-6", "rhel-9-0", "centos-9-stream")
```
Comment (garrett) [inline test/check-machines-migrate:93]: Either `doesn't` needs to be used (and the single tick quotes need to be double tick quotes) or we expand the word to `does not`, which is what I did in this file. I think it's just text to explain what's happening in the test, so it's fine to change, correct? There are several other occurrences in this file.

File: test/check-machines-migrate [hunk 2]
```diff
@@ -158,7 +158,7 @@ class TestMachinesMigration(VirtualMachinesCase):
 
         if fail == "uri":
             # test failure for syntactically incorrect destination uri
-            b.set_input_text("#dest-uri-input", "qemu+ssh://root@machine2/sistem")
+            b.set_input_text("#dest-uri-input", "qemu+ssh://root@machine2/system")
```
Comment (KKoukiou) [inline test/check-machines-migrate:161]: This is used as expected failure.

**Review by KKoukiou — CHANGES_REQUESTED** [review]

File: test/check-machines-migrate (hunk 1 above)
Comment (KKoukiou) [reply test/check-machines-migrate:93]: This is fine to change indeed.

File: test/check-machines-migrate (hunk 2 above)
Comment (garrett) [inline test/check-machines-migrate:161]: Oh, is this intentionally a spelling error to fail? I guess I should back out this change.

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**